jar-infer/src/main/java/com/uber/nullaway/jarinfer/definitelyDerefedParamsDriver.java

```diff
@@ -0,0 +1,115 @@
+package com.uber.nullaway.jarinfer;
+
+import com.ibm.wala.cfg.ControlFlowGraph;
+import com.ibm.wala.classLoader.IMethod;
+import com.ibm.wala.examples.util.ExampleUtil;
+import com.ibm.wala.ipa.callgraph.AnalysisCache;
+import com.ibm.wala.ipa.callgraph.AnalysisCacheImpl;
+import com.ibm.wala.ipa.callgraph.AnalysisOptions;
+import com.ibm.wala.ipa.callgraph.AnalysisScope;
+import com.ibm.wala.ipa.callgraph.CallGraphBuilderCancelException;
+import com.ibm.wala.ipa.callgraph.impl.Everywhere;
+import com.ibm.wala.ipa.cha.ClassHierarchyException;
+import com.ibm.wala.ipa.cha.ClassHierarchyFactory;
+import com.ibm.wala.ipa.cha.IClassHierarchy;
+import com.ibm.wala.ssa.IR;
+import com.ibm.wala.ssa.ISSABasicBlock;
+import com.ibm.wala.ssa.SSAInstruction;
+import com.ibm.wala.types.MethodReference;
+import com.ibm.wala.util.config.AnalysisScopeReader;
+import com.ibm.wala.util.io.CommandLine;
+import com.ibm.wala.util.strings.Atom;
+import com.ibm.wala.util.strings.ImmutableByteArray;
+import com.ibm.wala.util.strings.UTF8Convert;
+import com.ibm.wala.util.warnings.Warnings;
+import java.io.IOException;
+import java.util.*;
+import org.junit.Assert;
+
+/*
+ * Driver for running {@link definitelyDerefedParams}
+ */
+public class definitelyDerefedParamsDriver {
+
+  public static final String SCOPE_FILE_PATH = "./tests/tests.scope.txt";
+  public static final String SCOPE_MAIN_CLASS = "Ltoys/Main";
```

Comment:
Maybe something like `DEFAULT_SCOPE_FILE_PATH` and `DEFAULT_SCOPE_MAIN_CLASS` is a better name for these two?